packages/scripts/src/createView/scrubbed_messages_stats/daily.ts

```diff
@@ -0,0 +1,95 @@
+/**
+  Creates views with aggregated analytics stats. This is useful for things
+  like our dashboard which charts historical usage.
+ */
+
+import "dotenv/config";
+import { Db, Document } from "mongodb-rag-core";
+import { ChatbotStats } from ".";
+
+export type ScrubbedMessageStatsForDay = Document &
+  ChatbotStats & {
+    date: Date;
+    year: number;
+    dayOfYear: number;
+    month: number;
+    dayOfMonth: number;
+    week: number;
+    dayOfWeek: number;
+  };
+
+export function createScrubbedMessageStatsForDayView({
```

Comment:
rather than 3 separate files each w separate function, can we combine all the these functions as methods under 1 interface to reduce duplication of code

Reply:
Updated to use a "`makeCreator()`" pattern